Project does not install anymore (erlpack issue), need update - help!

When I `npm install`, I am getting an error about erlpack. I believe this is because the project depends on a forked version of `discord/erlpack` : `hammerandchisel/erlpack` that is probably not up-to-date. Also see https://github.com/discord/erlpack/issues/34

```
$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: erlpack@0.1.3
npm ERR! node_modules/erlpack
npm ERR!   erlpack@"github:hammerandchisel/erlpack" from botkit-discord@0.1.3
npm ERR!   node_modules/botkit-discord
npm ERR!     botkit-discord@"^0.1.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional erlpack@"discordapp/erlpack" from discord.js@11.6.4
npm ERR! node_modules/discord.js
npm ERR!   discord.js@"^11.4.2" from botkit-discord@0.1.3
npm ERR!   node_modules/botkit-discord
npm ERR!     botkit-discord@"^0.1.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/yann/.npm/eresolve-report.txt for a full report.
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "botkit": "^0.6.21",
     "clone-deep": "^4.0.1",
     "discord.js": "^11.4.2",
-    "erlpack": "github:hammerandchisel/erlpack",
+    "erlpack": "^0.1.3",
     "lodash.omit": "^4.5.0",
     "uws": "^99.0.0"
   },

